core/inc/knoting/forward_renderer.h

```diff
@@ -1,12 +1,28 @@
 #pragma once
 
-#include <subsystem.h>
+#include <knoting/log.h>
+#include <knoting/subsystem.h>
+#include <glm/matrix.hpp>
+#include <glm/vec3.hpp>
 
+#include <bgfx/bgfx.h>
+#include <knoting/mesh.h>
+#include <knoting/shader_program.h>
+#include <knoting/texture.h>
+
+#include <glm/gtc/matrix_transform.hpp>
+#include <glm/gtx/euler_angles.hpp>
```

Comment:
`#include <glm/gtc/matrix_transform.hpp>` is already in the `types.h`, but will add `#include <glm/gtx/euler_angles.hpp>` in the merge.